src/Beberlei/Metrics/Collector/Null.php

```diff
@@ -13,6 +13,9 @@
 
 namespace Beberlei\Metrics\Collector;
 
+/**
+ * @deprecated for PHP7 compatibility, use NullCollector instead.
+ */
 class Null implements Collector
```

Comment:
May be, this one can extends the new one.
More over, what do you think about trigger a deprecation error ?